docs/website/layouts/partials/docs/hero.html

```diff
@@ -1,7 +1,17 @@
 <section class="hero">
   <div class="hero-body">
-    <p class="title is-size-1 is-size-2-mobile">
-      {{ .Title }}
-    </p>
+    <span class="title is-size-1 is-size-2-mobile">
+      {{ .Title }} 
+    </span>
+    <span class="icon has-text-white-bis is-size-4 is-size-5-mobile icon-edit">
+        {{ if .File.Path}}
+        {{ $.Scratch.Set "LogicalName" .File.LogicalName}}
+        {{ end }}
+        {{ with .Site.Params.GithubEdit}}
+        <a href='{{ . }}{{ $.Scratch.Get "LogicalName" }}' target="_blank"><i class="fab fa-github"></i><span>Edit</span></a>
+        {{ end }}
```

Comment:
I might be missing something, but isn't this the same as
```
<a href='{{ .Site.Params.GithubEdit }}{{ .File.LogicalName }}' target="_blank"><i class="fab fa-github"></i><span>Edit</span></a>
```
? Perhaps we should have this _not get output_ if there is no `.File.LogicalName`, but I'm not sure if that ever happens. 🤔